runners/google-cloud-dataflow-java/build.gradle

```diff
@@ -275,19 +266,6 @@ afterEvaluate {
   }
 }
 
-task printFnApiPipelineOptions {
```

Comment:
We may want to keep this task and replace `fnApiPipelineOptions` with `runnerV2PipelineOptions`to help people inspect pipeline options when they want to run single test.

Reply:
Do people use this? I don't understand how since it `dependsOn buildAndPushDockerContainer` which actually builds an ephemeral container image, uploads it to gcr, then deletes it:

```
> Task :runners:google-cloud-dataflow-java:printRunnerV2PipelineOptions
To run a Dataflow job with runner V2, add the following pipeline options to your command-line:
--runner=TestDataflowRunner --project=apache-beam-testing --region=us-central1 --tempRoot=gs://temp-storage-for-validates-runner-tests --workerHarnessContainerImage=us.gcr.io/apache-beam-testing/java-postcommit-it/java:20201104192242 --experiments=beam_fn_api,use_unified_worker

> Task :runners:google-cloud-dataflow-java:cleanUpDockerImages
Untagged: us.gcr.io/apache-beam-testing/java-postcommit-it/java:20201104192242
Untagged: us.gcr.io/apache-beam-testing/java-postcommit-it/java@sha256:19baebd7dc6a9c1db7e99b230dc306effc4b9ec61e7d8203166c2bc80145e883
Digests:
- us.gcr.io/apache-beam-testing/java-postcommit-it/java@sha256:19baebd7dc6a9c1db7e99b230dc306effc4b9ec61e7d8203166c2bc80145e883
  Associated tags:
 - 20201104192242
Tags:
- us.gcr.io/apache-beam-testing/java-postcommit-it/java:20201104192242
Deleted [us.gcr.io/apache-beam-testing/java-postcommit-it/java:20201104192242].
Deleted [us.gcr.io/apache-beam-testing/java-postcommit-it/java@sha256:19baebd7dc6a9c1db7e99b230dc306effc4b9ec61e7d8203166c2bc80145e883].

Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.6.1/userguide/command_line_interface.html#sec:command_line_warnings
```

Reply:
I'm not sure whether we have folks use it. It's helpful to show all options we are using to launch such a test. But it seems like a bug to clean up the docker image.

Reply:
Without cleaning it up we'll end up with an ever-growing list of them and unfortunately I don't see a way to set TTL on objects in gcr. Maybe we could not bother with that if we had a perioid cleanup, which gcloud seems to support, but I haven't investigated on this a lot.

I most often use jenkins to find out how a job is executed.

Reply:
I added it back and refactored a bit so users could print the options, then later delete the image manually (and I printed the delete commands to the terminal).